.github/workflows/sync_chromium_branches.yaml

```diff
@@ -0,0 +1,48 @@
+name: Sync Chromium branches
+on:
+  workflow_dispatch:
+  # Runs everyday.
+  schedule:
+    - cron:  '0 0 * * *'
+permissions: read-all
+jobs:
+  sync:
+    runs-on: [self-hosted, chrobalt-linux-runner]
+    strategy:
+      # https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast
```

Comment:
nit: You can remove this comment. I think the line below is self-explanatory and we use this in other workflows without the comment.